spec/features/consumer/authentication_spec.rb

```diff
@@ -104,8 +104,7 @@
             expect do
               click_reset_password_button
               expect(page).to have_reset_password
-            end.to enqueue_job Delayed::PerformableMethod
-            expect(Delayed::Job.last.payload_object.method_name).to eq(:send_reset_password_instructions_without_delay)
+            end.to enqueue_job ActionMailer::DeliveryJob
```

Comment:
We're still checking that clicking the "reset password" button sends an email. We're not diving into the specifics of what that email looks like, but I think it might be sufficient in this case. The interface for examining some of these things in tests changed substantially and in some cases it was more difficult to check the specifics.

Reply:
I added a commit :+1: